lib/sivel2_gen/concerns/models/conscaso.rb

```diff
@@ -80,7 +80,7 @@ module Conscaso
           scope :filtro_categoria_id, lambda { |id|
             where('caso_id IN (SELECT id_caso
```

Comment:
Como la vista puede dar una id o un arreglo de ids, aquí debería venir un if.
Si es un arreglo usar IN
Si es un entero usar =